app/components/certificate/description.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  request: Boolean;
+  cert: Certificate;
+}
+
+export default function Description(props: Props) {
+  return (
+    <Flex flexDirection="column" gap="3" marginTop="14">
+      <Heading as="h1" size="xl">
+        Certificate
+      </Heading>
+      <Text>
+        {props.request
+          ? "Certificate: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+          : "Initial: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"}
+      </Text>
+      {props.request ? (
+        <HStack gap="6" marginTop="2">
+          <VStack>
+            <Text fontWeight="bold">Created On</Text>
+            <Text>{props.cert.validFrom.toDateString()}</Text>
```

Comment:
Use `chakra.time` for a `<time>` element, and use the `datatime` attribute to set a machine readable value.

Reply:
I'm sorry I can't seem to find the document relating to `chakra.time` do you happen to have the link for it?